FraihmworkHealthAndIntegrityApi.yaml

```diff
@@ -1787,6 +2020,71 @@ components:
         - sourceId
         - description
         - timeOfBlocklist
+           
+    UnavailabilitySchedule:
```

Comment:
All the other data structures here are perfectly alphabetical! This should be moved to it's correct spot at the bottom

Reply:
moved at the bottom.